### ISSUE TITLE
Error: not a tracked temp dir: 

### ISSUE THREAD

**Issue by awatson1978** [meteor-velocity/html-reporter#60]
Got this fun little error when trying to deploy an app with ``html-reporter`` to ``*.meteor.com``.

````sh
Abigails-Mini-2:minimongo-table abigailwatson$ sudo meteor deploy minimongo-table
                                                                                   
/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/dev_bundle/lib/node_modules/fibers/future.js:173
						throw(ex);
						      ^
Error: not a tracked temp dir: /Users/abigailwatson/.meteor/packages/velocity_core/.0.4.5.falldp++os.osx.x86_64+web.browser+web.cordova
    at Error (<anonymous>)
    at Object.files.freeTempDir (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/files.js:544:11)
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/tropohouse.js:348:17
    at Array.forEach (native)
    at Function._.each._.forEach (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/dev_bundle/lib/node_modules/underscore/underscore.js:79:11)
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/tropohouse.js:347:11
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:352:18
    at [object Object]._.extend.withValue (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:345:34
    at [object Object]._.extend.withValue (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:343:23
    at [object Object]._.extend.withValue (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at Object.enterJob (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:317:26)
    at Object.download (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/tropohouse.js:303:20)
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/tropohouse.js:412:22
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:352:18
    at [object Object]._.extend.withValue (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:345:34
    at [object Object]._.extend.withValue (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:343:23
    at [object Object]._.extend.withValue (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at Object.enterJob (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:317:26)
    at [object Object]._.extend.downloadPackagesMissingFromMap (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/tropohouse.js:409:20)
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/project-context.js:600:25
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:352:18
    at [object Object]._.extend.withValue (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:345:34
    at [object Object]._.extend.withValue (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:343:23
    at [object Object]._.extend.withValue (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at Object.enterJob (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:317:26)
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/project-context.js:599:20
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:264:13
    at [object Object]._.extend.withValue (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:257:29
    at [object Object]._.extend.withValue (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:255:18
    at [object Object]._.extend.withValue (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:246:23
    at [object Object]._.extend.withValue (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at Object.capture (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:245:19)
    at Object.catalog.runAndRetryWithRefreshIfHelpful (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/catalog.js:92:31)
    at [object Object]._.extend._downloadMissingPackages (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/project-context.js:598:13)
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/project-context.js:210:35
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:352:18
    at [object Object]._.extend.withValue (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:345:34
    at [object Object]._.extend.withValue (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:343:23
    at [object Object]._.extend.withValue (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at Object.enterJob (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:317:26)
    at [object Object]._.extend._completeStagesThrough (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/project-context.js:200:18)
    at [object Object]._.extend.saveChangedMetadata (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/project-context.js:188:10)
    at [object Object]._.extend.prepareProjectForBuild (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/project-context.js:193:10)
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/commands.js:1078:20
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:264:13
    at [object Object]._.extend.withValue (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:257:29
    at [object Object]._.extend.withValue (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:255:18
    at [object Object]._.extend.withValue (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:246:23
    at [object Object]._.extend.withValue (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at Object.capture (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:245:19)
    at Object.main.captureAndExit (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/main.js:259:29)
    at Command.main.registerCommand.name [as func] (/Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/commands.js:1077:8)
    at /Users/abigailwatson/.meteor/packages/meteor-tool/.1.0.40.k2gvmu++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/main.js:1300:23
````

**Comment by lionellei** [comment]
Did you figure it out? I also got this error. Very annoying. It used to work though. And I have no idea how to debug it.

**Comment by M0rph3v5** [comment]
I had this error, after trying again it worked :/

**Comment by lionellei** [comment]
I still had this error, trying it again and again doesn't work. I deployed it to modulus and it worked, but deploying to meteor.com doesn't.

**Comment by 4scottt** [comment]
I've got the same problem, retrying doesn't help

**Comment by Sanjo** [comment]
I think this is a meteor bug. Maybe it has something to do with the debugOnly flag? Should be reported to  https://github.com/meteor/meteor/issues.

**Comment by AdamBrodzinski** [comment]
+1 

I removed the velocity testing packages and deployed, then did a `git add . && git checkout -f` to reset back to a normal state, then the deploy works as expected.

**Comment by nickytonline** [comment]
I'm getting the same errors as @awatson1978. Of course I can deploy by removing the package, but this is annoying to add and remove. Has anyone opened an issue on github for meteor as @sanjo suggests?

**Comment by fourseven** [comment]
Just did everyone, since it's breaking a package I maintain too :(

Does html-reporter have binary builds?

**Comment by hwillson** [comment]
Hi all - the "Error: not a tracked temp dir" issue appears to have been fixed in the devel branch. See https://github.com/meteor/meteor/commit/a03dc12be71d5cbeb0da6a9ee8146881193efefc (I've updated https://github.com/meteor/meteor/issues/3761 with more details).

**Comment by kiide** [comment]
+1

**Comment by Sanjo** [comment]
Fixed in Meteor 1.0.3.2.
